libraries/AP_RCProtocol/AP_RCProtocol_CRSF.h

```diff
@@ -161,6 +171,21 @@ class AP_RCProtocol_CRSF : public AP_RCProtocol_Backend {
         int8_t downlink_dnr; // ( db )
     } PACKED;
 
+    enum RFMode : uint8_t {
+        CRSF_RF_MODE_4HZ = 0,
+        CRSF_RF_MODE_50HZ,
+        CRSF_RF_MODE_150HZ,
+    };
```

Comment:
This won't work because I have to compare it with link status data coming from a crossfire link status frame which uses an uint8_t for rf_mode, guess it could be made to work with a lot of casting

Reply:
...done by using static_cast<>